src/sync/lookupApiUtils.js

```diff
@@ -210,6 +210,8 @@ export const processPatientResponse = response => {
       date_of_birth,
       nameInsuranceJoin,
       nameNotes,
+      nationality_ID,
+      ethnicity_ID,
```

Comment:
@joshxg we don't have `female` either. Would you be able to add that in to this PR? or should I make a new issue just for that..?

Reply:
nice! 
a1b3bf4